examples/deepcopy-gen/output_tests/maps/doc.go

```diff
@@ -22,14 +22,15 @@ package maps
 type Ttest struct {
 	Byte map[string]byte
 	//Int8    map[string]int8 //TODO: int8 becomes byte in SnippetWriter
-	Int16   map[string]int16
-	Int32   map[string]int32
-	Int64   map[string]int64
-	Uint8   map[string]uint8
-	Uint16  map[string]uint16
-	Uint32  map[string]uint32
-	Uint64  map[string]uint64
-	Float32 map[string]float32
-	Float64 map[string]float64
-	String  map[string]string
+	Int16     map[string]int16
+	Int32     map[string]int32
+	Int64     map[string]int64
+	Uint8     map[string]uint8
+	Uint16    map[string]uint16
+	Uint32    map[string]uint32
+	Uint64    map[string]uint64
+	Float32   map[string]float32
+	Float64   map[string]float64
+	String    map[string]string
+	StringPtr map[string]*string
```

Comment:
Hm, does `map[string]**string` work too?

Reply:
might as well add tests cases

Reply:
I doubt ** works. But how often have you seen ** in Go? With aliases this might get messy, if we come up with something more generic.

Reply:
If it doesn't work the deep copier needs to at least fail rather than write wrong code.

Reply:
(in a follow-up is fine)

Reply:
Created an issue https://github.com/kubernetes/gengo/issues/109

We create uncompilable code at least, though proper errors would be certainly much better (true for many of our generators).